packages/backend/src/services/drive/add-file.ts

```diff
@@ -366,9 +366,16 @@ export async function addFile({
 	//#region Check drive usage
 	if (user && !isLink) {
 		const usage = await DriveFiles.calcDriveUsageOf(user);
+		const u = await Users.findOneBy({ id: user.id });
 
 		const instance = await fetchMeta();
-		const driveCapacity = 1024 * 1024 * (Users.isLocalUser(user) ? instance.localDriveCapacityMb : instance.remoteDriveCapacityMb);
+		let driveCapacity = 1024 * 1024 * (Users.isLocalUser(user) ? instance.localDriveCapacityMb : instance.remoteDriveCapacityMb);
```

Comment:
It seems you are using the wrong unit, according to the calculation it is Mebibyte (MiB). Does Misskey already use the wrong units in other cases too?

Reply:
That line was only changed from `const` to `let`, and the file size calculation remained the same as in the original, with mebibytes used.